Source/Core/VideoCommon/FramebufferManager.cpp

```diff
@@ -493,6 +493,9 @@ void FramebufferManager::RefreshPeekCache()
         flush_command_buffer = true;
       }
     }
+
+    m_efb_depth_cache.valid = true;
```

Comment:
I should have asked this originally but why not call `PopulateEFBCache()` for these instead?  Put another way, are the depth/color efbs valid if the rest of the logic in `PopulateEFBCache()` doesn't occur?

Reply:
Yes, InvalidatePeekCache will set present = false for each tile.